static/js/components/CandidateAnnotationsList.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from "react";
```

Comment:
Again, I'm confused by the filename here and how to distinguish this from the other new file in this PR